nss-day-cohort-60/rare-python-server-tyrannical-tators
# Lh add tags server

Part of PR that will close ticket #11

# Added posttag.py to models and posttag to repository.py for def all & def create. Joined posttag to post so that information is nested. 

---


# Steps to Test

1. Fetch the lh-add-tags-server branch and switch to it
2.  Run the debugger & open postman
3. Make a GET request http://localhost:8088/posts
4. Verify that post information has correct category, user, and tag information nested
